README.rst

```diff
@@ -21,6 +21,22 @@ Flask RestPlus
    :alt: Join the chat at https://gitter.im/noirbizarre/flask-restplus
    :target: https://gitter.im/noirbizarre/flask-restplus?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
 
+**IMPORTANT NOTICE**: The community has decided to fork the project due to lack
+of answers from the original author. We have been discussing this eventuality
+for almost [a year](https://github.com/noirbizarre/flask-restplus/issues/593).
+
+Things evolved a bit since that discussion and a few of us have been granted
+maintainers access to the github project, but we've been lacking rights on the
+Pypi project. Hence we haven't been able to release any new version.
```

Comment:
Apologies for nitpicking, but a capital letter is missing: `Pypi -> PyPi` :sweat_smile: